### DISCUSSION TITLE
Has anyone else had a new button show up in HomeKit with latest plugin update?

### DISCUSSION THREAD

**Discussion by Yashrg** [iklein99/homebridge-smartthings#154] [Q&A] [answered]
Hello friends,

I updated my home bridge-smartthings plugin to v1.5.3 today and now I have a programmable switch showing up in the Home app. It showed up in the Dining room and has the name Dining room. 
I only have one z-wave dimmer switch, one motion sensor and two contact sensors in that room inside of smart things. So I don't understand why the plugin is showing this unconfigured button and the console saying: [Smartthings Plug (IK)] Missing button status from Dining room. 

Does anyone have any idea what is going on?

**Comment by iklein99** [comment, marked as answer]
I added support for buttons.  One of your devices is reporting that it supports the button capability.  You can always add the device to the ignore devices list if you don't want it loaded into home kit.

**Reply by Yashrg** [reply]
You're right, thanks for bringing my attention to this. The device is a GE/Jasco z-wave dimmer. In the smart things app, there is a new entry called Button and it currently says Standby. This device must have recently switched to using this [edge driver](https://github.com/philh30/ST-Edge-Drivers/tree/main/GE-Jasco/ge-zwave-switch) I had installed. 

Is the expected behavior that console sends the missing button status that I'm seeing? In case I do decide to leverage the double tap or triple tap feature.

**Reply by coryabooth** [reply]
I noticed the same issue myself.  All my wall switches are now duplicated.  One for normal operations and one for the single/double/triple tap.

Although it would be a bit of work and probably hit the max ignore device limit - how to ignore the device when the single accessory has the same name duplicated?![image](https://github.com/iklein99/homebridge-smartthings/assets/5266177/a95ac692-2636-4d2e-8e63-5fb12154e035)

**Comment by iklein99** [comment]
The buttons are mapped to a "Stateless Programmable Switch" device in Homebridge.  This device supports single-push, double-push and long-push events.  However it will not work unless you set up the webhook service since this is strictly an event-driven device.  You can subscribe to that if you want.  https://www.kleinstudios.net

**Reply by Yashrg** [reply]
Thanks for the explanation. After some playing around with the driver and the double tap capabilities within smartthings, I found the implementation finicky. So at this time I have switched to a different edge driver without button function.
